Packages/com.unity.inputsystem/InputSystem/Editor/Settings/InputSettingsProvider.cs

```diff
@@ -301,6 +310,8 @@ private void InitializeWithCurrentSettings()
                     EditorGUI.LabelField(rect, m_Settings.supportedDevices[index]);
                 }
             };
+
+            m_iOSProvider = new InputSettingsiOSProvider(m_SettingsObject);
```

Comment:
For now, this is probably okay. However, going forward, this won't work across platforms. E.g. with consoles, we can't do this. Preferably, this should be done using the new polymorphic serialization. So it'd just be a list of SerializableObjects. Would also be easy to make the UI extensible this way.